arches_querysets/models.py

```diff
@@ -542,6 +546,34 @@ def _apply_provisional_edit(
 
         return oldprovisionalvalue, newprovisionalvalue, provisional_edit_log_details
 
+    def set_aliased_data(self, node, node_value):
+        """Format node_value according to the self._as_representation flag and
+        set it on self.aliased_data."""
+        datatype_instance = DataTypeFactory().get_instance(node.datatype)
+        empty_values = (None, "", '{"url": "", "url_label": ""}')
```

Comment:
Is there a way to pull out this mapping? `'{"url": "", "url_label": ""}'` is a very specific thing, and the hard mapping isn't extendable.

Reply:
I think we need to bite the bullet and "fix" the display value of this datatype in a future version of arches.